### PR TITLE
[PATCH v2] Pool perf options

### PR DESCRIPTION
**Pull request by psavol**
Added -n and -t options. -N controls how many bursts are allocated before freeing events back into the pool. -T enables testing with packet pools.

### CODE REVIEW

File: test/performance/odp_pool_perf.c [hunk 1]
```diff
@@ -219,19 +239,34 @@ static int test_pool(void *arg)
...
 	for (rounds = 0; rounds < num_round; rounds++) {
-		num = odp_buffer_alloc_multi(pool, buf, max_burst);
+		num = 0;
+
+		for (i = 0; i < num_burst; i++) {
+			ret = odp_buffer_alloc_multi(pool, &buf[num],
+						     max_burst);
+			if (odp_unlikely(ret < 0)) {
+				printf("Error: Alloc failed. Round %u\n",
+				       rounds);
```
Comment (matiaselo) [inline test/performance/odp_pool_perf.c]: Previously allocated buffers should be freed before exit.

File: test/performance/odp_pool_perf.c [hunk 2]
```diff
@@ -153,13 +169,15 @@ static int create_pool(test_global_t *global)
...
+	uint32_t num_burst = test_options->num_burst;
 	uint32_t num_cpu   = test_options->num_cpu;
 
 	printf("\nPool performance test\n");
 	printf("  num cpu    %u\n", num_cpu);
 	printf("  num rounds %u\n", num_round);
 	printf("  num events %u\n", num_event);
-	printf("  max burst  %u\n\n", max_burst);
+	printf("  max burst  %u\n", max_burst);
+	printf("  num burst  %u\n\n", num_burst);
```
Comment (matiaselo) [inline test/performance/odp_pool_perf.c]: "num bursts"

File: test/performance/odp_pool_perf.c (hunk 1 above)
Comment (psavol) [reply test/performance/odp_pool_perf.c]: fixed

File: test/performance/odp_pool_perf.c (hunk 2 above)
Comment (psavol) [reply test/performance/odp_pool_perf.c]: fixed

**Pull request merged**